spec/rails_helper.rb

```diff
@@ -62,3 +62,10 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
```

Comment:
- [ ] While running the `bundle exec rspec` the errors are coming. I have attached the screenshot as well. 

![Screenshot from 2022-05-07 00-37-23](https://user-images.githubusercontent.com/26767383/167202565-062ecd4f-6fe6-4ae4-86fa-c4ff993f3467.png)

It is happening because you have not included the gem `gem 'shoulda-matchers', '~> 5.0'` to the gemfile. Please add it After line 73.